website/source/docs/guides/containers-guide.md

```diff
@@ -0,0 +1,328 @@
+$ consul members
+Node      Address          Status  Type    Build  Protocol  DC   Segment
+server-1  172.17.0.2:8301  alive   server  1.5.2  2         dc1  <all>
+client-1  172.17.0.3:8301  alive   client  1.5.2  2         dc1  <default>
+```
+
+### Stopping, Starting, and Restarting Containers
+
+The official Consul container supports stopping, starting, and restarting. To stop a container, run `docker stop`.
+
+```sh
+$ docker stop <container_id>
+```
+
+To start a container, run `docker start`.
+
+```sh
+$ docker start <container_id>
+```
+
+To do an in-memory reload, send a SIGHUP to the container.
+
+```sh
+$ docker kill --signal=HUP <container_id>
+```
+
+### Removing Servers from the Cluster
+
+As long as there are enough servers in the cluster to maintain [quorum](/docs/internals/consensus.html#deployment-table), Consul's [autopilot](/docs/guides/autopilot.html) feature will handle removing servers whose containers were stopped. Autopilot's default settings are already configured correctly. If you override them, make sure that the following [settings](/docs/agent/options.html#autopilot) are appropriate.
+
+* `cleanup_dead_servers` must be set to true to make sure that a stopped container is removed from the cluster.
+* `last_contact_threshold` should be reasonably small, so that dead servers are removed quickly.
+* `server_stabilization_time` should be sufficiently large (on the order of several seconds) so that unstable servers are not added to the cluster until they stabilize.
+
+If the container running the currently-elected Consul server leader is stopped, a leader election will be triggered. This event will cause a new Consul server in the cluster to assume leadership.
+
+When a previously stopped server container is restarted using `docker start <container_id>`,  and it is configured to obtain a new IP, autopilot will add it back to the set of Raft peers with the same node-id and the new IP address, after which it can participate as a server again.
+
+
+### Backing-up Data
+
+You can back-up your Consul cluster using the [consul snapshot](https://www.consul.io/docs/commands/snapshot.html) command. 
+
+```sh
+$ docker exec <container_id> consul snapshot save backup.snap
+```
+
+This will leave the `backup.snap` snapshot file inside of your container. If you are not saving your snapshot to a [persistent volume](https://docs.docker.com/storage/volumes/) then you will need to use `docker cp` to move your snapshot to a location outside of your container.
+
+```sh
+$ docker cp <container_id>:backup.snap ./ 
+```
+
+Users running the Consul Enterprise Docker containers can run the [consul snapshot agent](https://www.consul.io/docs/commands/snapshot/agent.html) to save backups automatically. Consul Enterprise's snapshot agent also allows you to save snapshots to Amazon S3 and Azure Blob Storage.
+
+### Environment Variables
+
+You can also add configuration by passing the configuration JSON via environment variable `CONSUL_LOCAL_CONFIG`. 
```

Comment:
Editing since this was already mentioned

```suggestion
You can add configuration by passing the configuration JSON via the environment variable `CONSUL_LOCAL_CONFIG`. 
```